pythonforandroid/recipes/python2/__init__.py

```diff
@@ -49,23 +47,26 @@ def build_arch(self, arch):
 
         # This should be safe to run every time
         info('Copying hostpython binary to targetpython folder')
-        shprint(sh.cp, self.ctx.hostpython,
-                join(self.ctx.get_python_install_dir(), 'bin', 'python.host'))
-        self.ctx.hostpython = join(self.ctx.get_python_install_dir(), 'bin', 'python.host')
+
+        if isfile(join(self.ctx.get_python_install_dir(), 'bin', 'python.host')):
```

Comment:
Am I right in reading that the only change here is skipping the copy if the target already exists? If so, it would be preferable to clean it up a little more and put only the sh.cp call in the if block.